### PR TITLE
TCM-300 | Improve splash screen

### PR DESCRIPTION
**Pull request by devender197**
### Summary

- [x] Splash image height is dynamic
- [x] Bottom View fixed size of 180 and bottom padding of 47dp

### Issue

[TCM-300](https://hyfnla.atlassian.net/browse/TCM-300)

### Video/Screenshot

| Before | After | Small Device |
|-----|---|---|
|Screenshot_1621116620|Screenshot_1621131900|ScreenShot|

### CODE REVIEW

File: app/src/main/res/layout/fragment_splash.xml [hunk 1]
```diff
@@ -44,45 +45,52 @@
...
-        android:layout_marginEnd="15dp"
-        android:fontFamily="@font/atcarquette_bold"
-        android:gravity="center"
-        android:padding="13dp"
-        android:text="@string/splash_email_button_label"
-        android:textColor="@color/blackRussian"
-        android:textSize="13sp"
-        app:backgroundColor="@color/greyLightExtra"
-        app:layout_constraintTop_toBottomOf="@id/lblContinue" />
+        <com.trx.consumer.common.CommonLabel
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_splash.xml:54]: Looks like we'll want to center this label between the image and email button, everything else looks good

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Just one design related comment

File: app/src/main/res/layout/fragment_splash.xml (hunk 1 above)
Comment (devender197) [reply app/src/main/res/layout/fragment_splash.xml:54]: [Done](https://github.com/hyfn/trx-consumer-android/pull/31/files#diff-054adad281abaee9e8321ae785ac650586c1b6876b46c7c7ad4f22b369b59349R59)

**Comment by devender197** [comment]
@VivekV95 Thank you. I have implemented the changes. Please let me know if any changes required.

**Comment by onodude** [comment]
thank you for the demos @devender197

**Review by onodude — APPROVED** [review]

**Comment by onodude** [comment]
thanks @devender197

**Pull request merged**